frontend/app/templates/components/nav-bar.hbs

```diff
@@ -38,7 +38,7 @@
       <dd.menu @align="right" as |ddm|>
         <ddm.item>
           <a role="button" class="dropdown-item" {{on "click" this.toggleEncryptableCreating}}>
-            {{t "encryptables.new.title"}}
+            {{t "encryptable/credential.new.title"}}
```

Comment:
```suggestion
            {{t "encryptable/credentials.new.title"}}
```
it's a bit confusing, I know ... but we use the term encryptable/credentials (plural)
since an encryptable/credentials can include multiple credentials (username, password, ...)
so you also say ... please enter your credentials to login in ...

Reply:
Yes, this makes sense! I changed it in the latest commit